packages/arb-bridge-peripherals/contracts/tokenbridge/libraries/ClonableBeaconProxy.sol

```diff
@@ -15,7 +15,7 @@ contract ClonableBeaconProxy is BeaconProxy {
 }
 
 contract BeaconProxyFactory is ProxySetter {
```

Comment:
We can't really make breaking changes to this contract.
It wasn't deployed behind a proxy to ensure `type(ClonableBeaconProxy).creationCode` wouldn't break through upgrades